app.json

```diff
@@ -13,6 +13,10 @@
       "description": "development environment",
       "value": "development"
     },
+    "CI": {
```

Comment:
Can you explain how this changes the webpack config? From my understanding of herkou, this is unrelated and may start incurring charges. 
> Each CI-enabled Pipeline is $10/month (prorated to the second).
https://devcenter.heroku.com/articles/heroku-ci#costs

Reply:
We have `env.CI` set to true in travis and we check for it in the webpack config here to determine if we use the thread-loader: https://github.com/cerner/terra-core/blob/master/packages/terra-site/webpack.config.js#L43

I was adding the same env var named `CI` here but didn't consider heroku might have a similarly named env var. It looks like the CI env var doesn't enable heroku's CI, but heroku does use the CI variable. 

Knowing this we should likely avoid using the `CI` env var in a way it isn't intended. Instead we could use the `NODE_ENV` var which is set here. https://github.com/cerner/terra-core/blob/master/app.json#L12-L14 though, off the top of my head, I'm not sure if webpack sets this to var `development` unless the production flag is passed. Might be better to just add a `heroku` env var and set that to true and then check `heroku` and `CI`.